src/main/java/za/co/neslotech/spring/trainspringbootinsight/model/Transaction.java

```diff
@@ -17,6 +18,7 @@ public class Transaction {
     private Long id;
 
     @Column(name = "transaction_date")
+    @JsonProperty("transaction_date")
```

Comment:
JSON conventions should follow Java conventions of using `camelCase` for property names.